.github/workflows/build_test_deploy.yml

```
@@ -747,95 +738,137 @@ jobs:
+              turbo run build-native --cache-dir=".turbo" -- --release --target armv7-unknown-linux-gnueabihf
+              arm-linux-gnueabihf-strip packages/next-swc/native/next-swc.*.node
+          - host: ubuntu-latest
+            target: aarch64-linux-android
+            build: |
+              export CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android24-clang"
+              export CC="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android24-clang"
+              export CXX="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android24-clang++"
+              export PATH="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin:${PATH}"
+              npm i -g @napi-rs/cli@2.4.4 turbo@1.0.28
+              turbo run build-native --cache-dir=".turbo" -- --release --target aarch64-linux-android
+              ${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-strip packages/next-swc/native/next-swc.*.node
+          - host: ubuntu-latest
+            target: armv7-linux-androideabi
+            build: |
+              export CARGO_TARGET_ARMV7_LINUX_ANDROIDEABI_LINKER="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi24-clang"
+              export CC="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi24-clang"
+              export CXX="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi24-clang++"
+              export PATH="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin:${PATH}"
+              npm i -g @napi-rs/cli@2.4.4 turbo@1.0.28
+              turbo run build-native --cache-dir=".turbo" -- --release --target armv7-linux-androideabi
+              ${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-strip packages/next-swc/native/next-swc.*.node
+          - host: ubuntu-latest
+            target: 'aarch64-unknown-linux-musl'
+            docker: ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-alpine
+            build: >-
+              set -e &&
+              npm i -g @napi-rs/cli@2.4.4 turbo@1.0.28 &&
+              rustup toolchain install nightly-2021-11-15 &&
+              rustup default nightly-2021-11-15 &&
+              rustup target add aarch64-unknown-linux-musl &&
+              turbo run build-native --cache-dir=".turbo" -- --release --target aarch64-unknown-linux-musl &&
+              llvm-strip -x packages/next-swc/native/next-swc.*.node
+          - host: windows-latest
+            target: 'aarch64-pc-windows-msvc'
+            build: |
+              npm i -g @napi-rs/cli@2.4.4 turbo@1.0.28
+              turbo run build-native --cache-dir=".turbo" -- --release --target aarch64-pc-windows-msvc
+    if: ${{ needs.build.outputs.isRelease == 'true' }}
```

Can we do a test run with this line commented out to ensure this builds as expected?
```suggestion
    # if: ${{ needs.build.outputs.isRelease == 'true' }}
```

---

https://github.com/vercel/next.js/runs/5242914905?check_suite_focus=true build passed